src/schema/objects/metadata.yaml

```diff
@@ -2868,60 +3036,322 @@ _LandmarkCoordinates:
 _MEGCoordSys:
   type: string
   enum:
-  - CTF
-  - ElektaNeuromag
-  - 4DBti
-  - KitYokogawa
-  - ChietiItab
-  - Other
+  - CTF:
+      name: CTF
+      description: |
+        ALS orientation and the origin between the ears.
+  - ElektaNeuromag:
+      name: Elekta Neuromag
+      description: |
+        RAS orientation and the origin between the ears.
+  - 4DBti:
+      name: 4D BTI
+      description: |
+        ALS orientation and the origin between the ears.
+  - KitYokogawa:
+      name: KIT/Yokogawa
+      description: |
+        ALS orientation and the origin between the ears.
+  - ChietiItab:
+      name: Chieti ITAB
+      description: |
+        RAS orientation and the origin between the ears.
+  - Other:
+      name: Other
+      description: |
+        Other coordinate system.
+_NonstandardTemplateCoordSys:
+  type: string
+  enum:
+  - individual:
+      name: individual
+      description: |
+        Participant specific anatomical space (for example derived from T1w and/or T2w images).
+        This coordinate system requires specifying an additional, participant-specific file to be fully defined.
+        In context of surfaces this space has been referred to as `fsnative`.
+
+        In order for this space to be interpretable, `SpatialReference` metadata MUST be provided.
+  - study:
+      name: study
+      description: |
+        Custom space defined using a group/study-specific template.
+        This coordinate system requires specifying an additional file to be fully defined.
+
+        In order for this space to be interpretable, `SpatialReference` metadata MUST be provided.
+_NonTemplateCoordSys:
+  type: string
+  enum:
+  - scanner:
+      name: scanner
+      description: |
+        The intrinsic coordinate system of the original image (the first entry of `RawSources`)
+        after reconstruction and conversion to NIfTI or equivalent for the case of surfaces and dual volume/surface files.
+
+        The `scanner` coordinate system is implicit and assumed by default if the derivative filename does not
+        define **any** `space-<label>`.
+        Please note that `space-scanner` SHOULD NOT be used,
+        it is mentioned in this specification to make its existence explicit.
```

Comment:
I'm not sure if these values should be enumerated here. I took them from https://bids-specification.readthedocs.io/en/stable/99-appendices/08-coordinate-systems.html#nonstandard-coordinate-system-identifiers and https://bids-specification.readthedocs.io/en/stable/99-appendices/08-coordinate-systems.html#non-template-coordinate-system-identifiers.